unlock-protocol.com/blog/version-1-2.md

```diff
@@ -0,0 +1,87 @@
+
+2. _A bonded curve lock_: a hook could increase the price of keys on a lock based on how many keys have been sold in the past. This can create bonding curves where early members end up paying less than late members, while also having the ability to re-sell their early membership at a higher price. You can easily use a lock as a [Token Curated Registry](https://medium.com/@ilovebagels/token-curated-registries-1-0-61a232f8dac7) where only members of the lock can change the data of the registry!
+
+In practice, there is an endless amount of possibilities here and we cannot wait to see what people are building.
+
+## Full refunds
+
+This has been a request by the [EthGlobal](https://unlock-protocol.com/blog/ethwaterloo-tickets/) team: they wanted the ability to easily refund all participants to the hackathon, if they checked-in. Until then, it required a 2 step process, where the lock owner would have to withdraw the funds from the lock and then send them to the participants from their own address. Now, it can be done through the lock itself, making refunds very easy ;)
+
+## Risk Free trials
+
+The keys to a lock can be "burned" by their owner, triggering a pro-rated refund, based on how long the key has been used. Lock creators now have a way to define a period during which the key owner can request a _full refund_, making risk free trials possible.
+
+## Tips
+
+When we introduced our donation locks, several users asked us about the ability to "pay more" than the required price. Up until now, it was actually not possible to submit the actual amount that a consumer wants to pay for a membership key. With this new version, whoever purchases a key actually submits the amount they want to pay (greater than or equal to the price set by the creator).
+
+## Partial sharing
+
+I am very excited about this one: partial sharing of a membership. [Unlock keys are non-fungible](/blog/non-fungible-tokens-betaworks/), but... they are time based, and you can now split a membership and send some of your time to someone else! This has a lot of very interesting implications. For example, members can now share "paywall-ed" links along with a few minutes worth of memberships, in order to let someone else read the content!
+
+# Building on top of Unlock
+
+Our smart contracts are the "lowest" point of integration. Once a lock has been deployed, its functionality is immutable, which means that other application can start integrating with it in a safe way as it's API will never change. We publish npm modules which includes interfaces to our smart contracts. Here is our [npm module for version 1.2](https://www.npmjs.com/package/unlock-abi-1-2). It's the "raw" API for locks of that version.
```

Comment:
Drop the "s" in  "includes":
We publish npm modules which include interfaces to our smart contracts.